recipe/meta.yaml

```
@@ -10,32 +10,45 @@ source:
   sha256: dd5be4a72d8a2995c3f583cf82bf3cd1a9544cfdabf2d22595b67aff07349666
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 1
+  number: 0
```

```suggestion
  number: 0
```
Leaving similar comment to jsonpath regarding the version and build number. Are these initializations so it doesn't matter that the version remains the same with and the build number is 0?

---

Heh, I went through the same process[1] as for the other package! 😄 

Again, because we hadn't published this before I set the build number to 0.  I thought that was the policy, I could easily be wrong.  I'm not sure it hurts anyone either way.  🤔 

[1] I ran my tool and that's what it suggested!  😆

---

If it's the initial publishing of the package and therefore there's no version that already exists, then yeah this is fine. That's my fault for not realizing it was the first of its kind. Lol